dot/network/gossip_test.go

```diff
@@ -25,6 +25,10 @@ import (
 
 // test gossip messages to connected peers
 func TestGossip(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping TestGossip")
+	}
+
```

Comment:
Is this being skipped because noise was added?  Or another reason?  Should we add a todo/issue?

Reply:
unrelated, just seemed to be failing on CI a lot recently. yeah I'll make an issue

Reply:
Ok, thanks.